docs/components/modeler/web-modeler/additional-features/call-activity-linking.md

```diff
@@ -0,0 +1,27 @@
+---
+id: call-activity-linking
+title: Call activity linking
+description: Call activity linking
+---
+
+You can use one of the following approaches to link a [call activity](/components/modeler/bpmn/call-activities/call-activities.md) to another process.
+
+### Using the link button
+
+If you select a call activity, a link button appears at the bottom right. Clicking on the button opens a dialog from which you can choose any diagram from the same project. Click on the `Link` button after selecting the diagram. This will create the link by copying the process ID of the diagram to the `Called element` section in the properties panel.
```

Comment:
...create the link by _pasting_ the process ID of the diagram to...

Reply:
Howevery, this is a very transparent explanation of how this is achieved, great!

Reply:
If we think in terms of the `copy` and `paste` operations, the word _pasting_ makes sense.

But when explaining in English, isn't to correct to say _copying_?

Reply:
I am note 100% sure, this is what my grammar check gave me

Reply:
How is it written is correct. "Copied to" is essentially "brought over to/duplicated in".